arch/arm/configs/tsimx6_defconfig

```diff
@@ -647,61 +673,27 @@ CONFIG_RSI_91X=m
 CONFIG_WL12XX=m
 CONFIG_WLCORE_SDIO=m
 # CONFIG_WILINK_PLATFORM_DATA is not set
+CONFIG_USB_ZD1201=m
 CONFIG_ZD1211RW=m
 CONFIG_QTNFMAC_PCIE=m
-CONFIG_USB_NET_RNDIS_WLAN=m
-CONFIG_VIRT_WIFI=m
+# CONFIG_INPUT_LEDS is not set
 CONFIG_INPUT_MOUSEDEV=m
-CONFIG_INPUT_JOYDEV=m
 CONFIG_INPUT_EVDEV=y
+# CONFIG_KEYBOARD_ATKBD is not set
 CONFIG_KEYBOARD_GPIO=y
-CONFIG_KEYBOARD_SNVS_PWRKEY=y
```

Comment:
I don't think this is an issue on imx6, but, be careful removing this as this changes the input event numbering. I know this is an issue in the imx6ul, specifically ts7553v2 for the keypad demos. But I don't believe this is an issue to remove here.